core/frontend/src/tile/TileTree.ts

```diff
@@ -159,4 +160,28 @@ export abstract class TileTree {
   public accumulateTransformedRange(range: Range3d, matrix: Matrix4d, location: Transform, frustumPlanes?: FrustumPlanes): void {
     this.rootTile.extendRangeForContent(range, matrix, location, frustumPlanes);
   }
+
+  /**
+   * Return the transform from the tile tree's coordinate space to [ECEF] (https://en.wikipedia.org/wiki/ECEF) (Earth Centered Earth Fixed) coordinates.

```

Comment:
Link won't display correctly with space between `]` and `(`.

Reply:
Fixed link.